tests/Cms/System/LicenseTest.php

```diff
@@ -11,6 +11,8 @@
  */
 class LicenseTest extends TestCase
 {
+	public const TMP = KIRBY_TMP_DIR . '/Cms.License';
```

Comment:
Are we not doing `/Cms/License` as this would have to create nested folders? Otherwise, I think that would be preferable over the dot notation.

Reply:
Yes, with nested folders we can't clean them up as easily. The entire `tmp` directory is removed anyway after the test runs and I even considered just using `KIRBY_TMP_DIR` everywhere, but we have some tests that don't fully clean up after themselves, so separating each test class is safest – and that requires using one directory level, so no slashes.